recipes/fsleyes-widgets/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "fsleyes-widgets" %}
+{% set version = "0.6.1" %}
+{% set sha256 = "9d60fefbb8f8aec467aac0057df3eea91f553bcb5a601ed649a9f6d1628b017a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pip
+    - sphinx 1.6.*
```

Comment:
These seem like runtime deps not build time

Reply:
No, they're build deps - I have a custom setuptools command for building API docs (e.g. `python setup.py doc`). Obviously this command will not be invoked when building the package though. Happy to remove them if you want me to!